crates/btc-notify/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "btc-notify"
+version = "0.1.0"
+edition = "2021"
+
+[lints]
+rust.missing_debug_implementations = "warn"
+rust.missing_docs = "warn"
+rust.rust_2018_idioms = { level = "deny", priority = -1 }
+rust.unexpected_cfgs = { level = "warn", check-cfg = [
+  'cfg(coverage, coverage_nightly)',
+] }
+rust.unreachable_pub = "warn"
+rust.unused_crate_dependencies = "deny"
+rust.unused_must_use = "deny"
+rustdoc.all = "warn"
+
+[dependencies]
+bitcoin.workspace = true
+bitcoincore-zmq = { version = "1.5.2", features = ["async"] }
+futures = "0.3.31"
```

Comment:
This might be worth to move to the root-level `Cargo.toml`. I bet that other crates (especially the `bin/` ones) might use it.